pkg/plugins/policies/meshhttproute/api/v1alpha1/meshhttproute.go

```diff
@@ -0,0 +1,68 @@
+// +kubebuilder:object:generate=true
+package v1alpha1
+
+import (
+	common_api "github.com/kumahq/kuma/api/common/v1alpha1"
+)
+
+// MeshHTTPRoute
+// +kuma:policy:skip_registration=true
+//
+// This policy defines its own `GetDefault` method so that it can have the given
+// structure for deserialization but still use the generic policy merging
+// machinery.
+//
+// +kuma:policy:skip_get_default=true
+type MeshHTTPRoute struct {
+	// TargetRef is a reference to the resource the policy takes an effect on.
+	// The resource could be either a real store object or virtual resource
+	// defined inplace.
+	TargetRef common_api.TargetRef `json:"targetRef,omitempty"`
+
+	// To matches destination services of requests and holds configuration.
+	To []To `json:"to,omitempty"`
+}
+
+// This type exists to avoid a NPE in controller-tools
+type Empty struct{}
+
+type To struct {
+	// TargetRef is a reference to the resource that represents a group of
+	// request destinations.
+	TargetRef common_api.TargetRef `json:"targetRef,omitempty"`
+	// Rules contains the routing rules applies to a combination of top-level
+	// targetRef and the targetRef in this entry.
+	Rules []Rule `json:"rules,omitempty"`
+
+	// Default is here to satisfy the many assumptions of the policy generation
+	// code that it exists. It isn't included in the schema.
+	Default Empty `json:",omitempty"`
```

Comment:
There's no way to adapt the policy generation?

Reply:
Probably, but it's not worth refactoring IMO. This `Default` only shows up in the code, not the CRDs or serialized resources.